modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc

```diff
@@ -377,3 +385,152 @@ Scanner output rules are used in daemon/socket mode.
 . A file size limit applies both to background jobs and to file uploads.
 . After installing ClamAV and the related tools, you will have two configuration files: `/etc/freshclam.conf` and `/etc/clamd.d/scan.conf`. 
 . We recommend that you enable verbose logging in both `clamd.conf` and `freshclam.conf` until you get any kinks with your ClamAV installation worked out
+
+
+== ICAP
+
+{icap-url}[ICAP] is an open standard supported by many antivirus products. With the release of the _Anti-Virus_ app 1.0.0, other virus scanners beside ClamAV can be used via ICAP if you are running it on an ownCloud Enterprise Edition. Currently the only tested and supported virus scanners, besides ClamAV, are _Kaspersky ScanEngine_ and _McAfee Antivirus_ although far more products might simply work.
+
+=== Installation
+
+- Install the {oc-marketplace-url}/apps/files_antivirus[Anti-Virus app] from the ownCloud marketplace. Alternatively, use this occ command:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} market:install files_antivirus
+----
+
+- Enable the app as admin in ownCloud under menu:[Settings > Apps] in the category `Security` or with the following occ command:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} app:enable files_antivirus
+----
+
+
+=== General Configuration
+
+ICAP can be configured via the Web interface as admin user or via xref:occ_command.adoc[occ commands].
+
+==== Antivirus Configuration Panel
+
+Log in to ownCloud as admin via the Web interface and go to menu:Admin[Settings > Security] and you'll see the Antivirus Configuration dialog.
+
+image:configuration/server/anti-virus-panel.png[Security Settings, width:80%]
+
+Enter the desired values:
+
+* `Mode`: Set to `Daemon (ICAP)`.
+
+* `Host`: Enter the IP addess of your ICAP server.
+
+* `Port`: Specify the port number (default 1344).
+
+* `Stream Length`: Set the length of streams sent to the ICAP server in bytes.
+
+* `File size limit`: If you want to limit the file size, enter the maximum value in bytes. Default is no limit (-1).
+
+* `ICAP request service`: Select the antivirus software you want to use: avscan for ClamAV, req for Kaspersky ScanEngine or `wwreqmod` for McAfee.
+
+* `ICAP response header holding the virus information`: Use `X-Infection-Found` for ClamAV (avscan) and `X-Virus-ID` for KAV (req). McAfee doesn't offer response headers.
+
+* `When infected files were found during a background scan`: Specify what to do with the flagged files. Possible values: `Delete file` or `Only log`.
```

Comment:
```suggestion
`Mode`::
Set to `Daemon (ICAP)`.

`Host`::
Enter the IP addess of your ICAP server.

`Port`::
Specify the port number (default 1344).

`Stream Length`::
Set the length of streams sent to the ICAP server in bytes.

`File size limit`::
If you want to limit the file size, enter the maximum value in bytes. Default is no limit (-1).

`ICAP request service`::
Select the antivirus software you want to use: avscan for ClamAV, req for Kaspersky ScanEngine or `wwreqmod` for McAfee.

`ICAP response header holding the virus information`::
Use `X-Infection-Found` for ClamAV (avscan) and `X-Virus-ID` for KAV (req). McAfee doesn't offer response headers.

`When infected files were found during a background scan`::
Specify what to do with the flagged files. Possible values: `Delete file` or `Only log`.
```
This renders a bit better, see screenshot:
![image](https://user-images.githubusercontent.com/3321281/115008283-3b837800-9eab-11eb-8ce3-c6bb330db90a.png)